apps/onboarding/src/gateway/gateway.service.spec.ts

```diff
@@ -1,12 +1,16 @@
 import { Test, TestingModule } from '@nestjs/testing'
+import { HttpModule, HttpService } from '@nestjs/common'
 import { GatewayService } from './gateway.service'
+import { AppModule } from '../app.module'
+import { CaptchaService } from './captcha/captcha.service'
 
 describe('GatewayService', () => {
   let service: GatewayService
 
   beforeEach(async () => {
     const module: TestingModule = await Test.createTestingModule({
-      providers: [GatewayService]
+      imports: [HttpModule, AppModule],
```

Comment:
let's remove the dependency on `AppModule` here. we shouldn't import the `AppModule` in unit tests but actually use the `createTestingModule` to setup a lighter module that only includes what we need to test current service.